src/main/scala/org/broadinstitute/hail/driver/ExportVCF.scala

```diff
@@ -205,7 +216,46 @@ object ExportVCF extends Command {
           sb.append("GT:AD:DP:GQ:PL")
         gs.foreach { g =>
           sb += '\t'
-          sb.append(g)
+
+          sb.append(g.gt.map { gt =>
+            val p = Genotype.gtPair(gt)
+            s"${ p.j }/${ p.k }"
+          }.getOrElse("./."))
+
+          (g.ad, g.dp, g.gq,
+            if (g.isDosage)
+              g.dosage.map(Left(_))
+            else
+              g.pl.map(Right(_))) match {
+            case (None, None, None, None) =>
+            case (Some(ad), None, None, None) =>
+              sb += ':'
+              append(sb, ad)
+            case (ad, Some(dp), None, None) =>
+              sb += ':'
+              appendOption(sb, ad)
+              sb += ':'
+              sb.append(dp) // Int
+            case (ad, dp, Some(gq), None) =>
+              sb += ':'
+              appendOption(sb, ad)
+              sb += ':'
+              sb.append(dp.getOrElse('.'))
```

Comment:
This is also boxing and calling the version of `append` that takes `Any` since the common supertype of `Int` and `Char` is `AnyVal`:

```
scala> val o: Option[Int] = Some(5)
o: Option[Int] = Some(5)

scala> o.getOrElse('.')
res5: AnyVal = 5
```